clients/stellar-relay-lib/src/tests/mod.rs

```diff
@@ -19,9 +19,10 @@ async fn stellar_overlay_connect_and_listen_connect_message() {
 			.unwrap();
 
 	let node_info = NodeInfo::new(19, 25, 23, "v19.5.0".to_string(), &PUBLIC_NETWORK);
-	let cfg = ConnectionInfo::new(TIER_1_VALIDATOR_IP_PUBLIC, 11625, secret, 0, false, true, false);
+	let conn_info =
+		ConnectionInfo::new(TIER_1_VALIDATOR_IP_PUBLIC, 11625, secret, 0, false, true, false);
```

Comment:
Can we change these so that they take the info from the `clients/vault/resources/config/testnet/stellar_relay_config_sdftest1.json` file?

Reply:
see [bcd60ec](https://github.com/pendulum-chain/spacewalk/pull/269/commits/bcd60ecf122aadd50ed5529a268439ea81b4e454)

Reply:
Perfect 👍